hasher-matcher-actioner/hmalib/lambdas/api/api_root.py

```diff
@@ -378,10 +387,13 @@ def get_dashboard_actions() -> DashboardCount:
 
 
 def get_dashboard_signals() -> DashboardCount:
+    """
+    note: SIGNALS COUNT != NUMBER OF UNQUIE HASHES
+    """
```

Comment:
Can we describe what it is (instead of what its not)

Reply:
none of the functions like this one have docstrings because they are pretty straight forward this is just a heads up for someone coming back that might assume.

Reply:
Signals is the only one that seems non obvious to me. "Hashes" and "matches" we elsewhere in the code and documentation "Signals" I haven't seen so I think a docstring here is more necessary

Reply:
I don't disagree but going to leave it for now if that is alright with you. we're approaching the need for a glossary as is and I'd rather be more consistent in clarification around 'Signals' overall.